### ISSUE TITLE
gifv won't play

### ISSUE THREAD

**Issue by mgillard1976** [Eloston/disable-html5-autoplay#96]
Even opening the controls and hitting play I can't play a gifv. A new blocked instance shows up every time.

**Comment by Eloston** [comment]
Huh. Do you have a link for me to test this out?

**Comment by mgillard1976** [comment]
[Here's one](http://i.imgur.com/K98wsES.gifv)

Unless I "Disable Nothing" I can't get this to play.

I'm on a mac, OS 10.11.6

**Comment by Iliyass** [comment]
same, all 9gag gifs doesn't till I disable the plugin

**Comment by mminkus** [comment]
Yeah, all imgur videos are broken. I don't know why. Clicking on it to play doesn't work. I just whitelisted imgur.com and i.imgur.com and it made the problem go away as a workaround.

Here's some examples though:

https://i.imgur.com/uswumoy.gifv
https://i.imgur.com/uswumoy
https://i.imgur.com/uswumoy.jpg?r    (but this works?)

**Comment by tedder** [comment]
Yeah, it'd be nice to be able to allow gifs- not on a given site, but perhaps by mime type?

**Comment by rosorrentino** [comment]
Same with Facebook gif button...
